closed/autoconf/custom-hook.m4

```diff
@@ -60,7 +60,7 @@ AC_DEFUN([OPENJ9_CONFIGURE_CMAKE],
     ],
     [
       case "$OPENJ9_PLATFORM_CODE" in
-        oa64|xa64)
```

Comment:
This doesn't affect the XL builds, is that intensional?

Reply:
`OPENJ9_PLATFORM_CODE` is independent of normal/XL.

Reply:
When I look at the nightly builds for xlinux, it seems the normal build is using cmake and the XL build is not.

Reply:
are you sure you are looking at jdk11? Looks like cmake to me https://ci.eclipse.org/openj9/job/Build_JDK11_x86-64_linux_xl_Nightly/423/console

Reply:
Yes, my mistake. I was looking at jdk11 but at a plinux instead of xlinux build.